app/src/main/java/org/shadowice/flocke/andotp/Activities/MainActivity.java

```diff
@@ -378,6 +383,7 @@ public void run() {
             });
         super.onPause();
         stopUpdater();
+        countDownTimer.cancel();
```

Comment:
We need a ``if (countDownTimer != null)`` here otherwise it will crash of the option is not enabled and the MainActivity gets paused. I will merge the request and fix it myself since it's such a small thing so we can get it done quicker.